addons/zylann.editor_debugger/dock.gd

```diff
@@ -56,10 +56,11 @@ func get_tree_view() -> Tree:
 
 func _ready() -> void:
 	_popup_menu.clear()
-	for id in _popup_action_names:
-		_popup_menu.add_item(_popup_action_names[id].title, id)
+	for id: int in _popup_action_names:
```

Comment:
The syntax `id: int` doesn't work in Godot 4.1. That means the plugin will have to mention that it can only be used with 4.2 or later from now on.

Reply:
I'll double check, but I think this syntax really only causes an issue with the Inferred Declaration warning, which at least in 4.2 is a little bit broken anyway to the point that I doubt almost anyone is using it, and would not be unreasonable to just disregard. If that's true then I'll undo these for loop changes.